lametro/views.py

```diff
@@ -423,7 +424,7 @@ def get_context_data(self, *args, **kwargs):
             WHERE m.organization_id='ocd-organization/42e23f04-de78-436a-bec5-ab240c1b977c'
             AND m.end_date >= '{0}'
             AND m.person_id <> 'ocd-person/912c8ddf-8d04-4f7f-847d-2daf84e096e2'
-            GROUP BY p.ocd_id, p.name
+            GROUP BY p.ocd_id, p.name, m.extras
```

Comment:
Maybe this is a nonsensical question but if a person had more than one post, would this return 2 rows for that person because of the `extras` field being part of the `GROUP BY`? Does that even happen? Does it matter?

Reply:
that's true, but the same person should not have more than one post – so, if that happens, then something is wrong with the data upstream.

Reply:
it seems to me like the Councilmatic site should not mask such data errors? what do you think?

Reply:
Yeah that seems reasonable. I'm not really sure what else we'd do in this case.